package/src/prmon.cpp

```diff
@@ -80,6 +81,9 @@ int ProcessMonitor(const pid_t mpid, const std::string filename,
   std::stringstream json_snapshot_file;
   json_snapshot_file << json_summary_file << "_snapshot";
 
+  // Add prmon version file to the JSON
+  json_summary["Version"] = prmon_VERSION;
```

Comment:
How about a new section for this type of information, i.e.,

```c++
 json_summary["prmon"]["Version"] = prmon_VERSION;
```

Reply:
That's a good idea, now we have a dedicated block.